.gitignore

```diff
@@ -1,2 +1,2 @@
 .DS_Store
-/wordpress*
+/wordpress*
```

Comment:
A very minor point, but if you have a chance, it'd be good to restore the newline at the end of this file.